src/pages/console/authz/permissions.mdx

```diff
@@ -44,4 +44,4 @@ If you want your data model to be publicly accessible, switch to API_KEY or IAM
 1. Using the *Books* data model that you created in the [Create a data model example](/console/data/data-model#Create-a-data-model-example), set the authorization mode to **API Key**. 
 2. In the **Model** pane on the right, expand the **Anyone** window. Choose **Read** to specify that any signed in user has read access to the data in the *Book* model. The settings look as follows.
 
-![Public authorization rules on a GraphQL data model](/images/console/7_publicauthreadonly.png)
+![](/images/console/7_publicauthreadonly.png)
```

Comment:
Since we're making the image decorative, I think we shouldn't have the "The settings look as follows" sentence above it. 

If it's decorative then it means it can be removed and the content wouldn't be missing anything. So I think it makes sense to not mention it at all.

Hopefully that makes sense